assets/css/main.css

```diff
@@ -48,6 +47,23 @@ ul li {
   text-align: left;
  }
 
+.white-bg{
```

Comment:
Since you're editing CSS files, can you change the version number in the html file.